src/Features/CSharp/Portable/SignatureHelp/ConstructorInitializerSignatureHelpProvider.cs

```diff
@@ -96,16 +101,15 @@ private static bool IsArgumentListToken(ConstructorInitializerSyntax expression,
                              .Sort(semanticModel, constructorInitializer.SpanStart);
 
             if (!constructors.Any())
-            {
                 return null;
-            }
 
             // guess the best candidate if needed and determine parameter index
             var arguments = constructorInitializer.ArgumentList.Arguments;
             var candidates = semanticModel.GetSymbolInfo(constructorInitializer, cancellationToken).Symbol is IMethodSymbol exactSymbol
                 ? ImmutableArray.Create(exactSymbol)
                 : constructors;
-            LightweightOverloadResolution.RefineOverloadAndPickParameter(document, position, semanticModel, candidates, arguments, out var currentSymbol, out var parameterIndex);
+            LightweightOverloadResolution.RefineOverloadAndPickParameter(
+                document, position, semanticModel, candidates, arguments, out var currentSymbol, out var parameterIndexOverride);
```

Comment:
renamed parameterIndex to parameterIndexOverride to make it clearer what thsi is trying to do.